src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/PropertyGridInternal/PropertyGridView.DropDownHolder.DropDownHolderAccessibleObjectTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Reflection;
+using WinForms.Common.Tests;
+using Xunit;
+using static Interop;
+
+namespace System.Windows.Forms.PropertyGridInternal.Tests
+{
+    public class DropDownHolderAccessibleObjectTests
+    {
+        [WinFormsFact]
+        public void DropDownHolder_AccessibilityObject_Constructor_initializes_fields()
+        {
+            using PropertyGrid propertyGrid = new PropertyGrid();
+
+            PropertyGridView propertyGridView = (PropertyGridView)propertyGrid.TestAccessor().Dynamic._gridView;
```

Comment:
We now have `PropertyGridTestAccessor`, and we have tests that show how to use it, e.g.  `PropertyDescriptorGridEntryAccessibleObject_ExpandCollapseState_reflects_ExpandablePropertyState`
```suggestion
            PropertyGridView propertyGridView = propertyGrid.TestAccessor().GridView;
```

The same applies everywhere else